README.md

```diff
@@ -1,6 +1,12 @@
 # Hello Microverse
```

Comment:
- [ ] The `.gitignore` should include the `node_modules` directory, kindly consider including it in your `.gitignore` file so it will not be pushed to Github by accident (in future projects you will use node_modules)

Reply:
I added node_modules to .gitignore.